### PR TITLE
Pop-up modal 

### PR DESCRIPTION
**Pull request by zemola**
# MY PORTFOLIO POP-UP MODAL 

Collaborated with my coding partner in developing a pop-up modal for my portfolio putting in mind the following criteria.

1. On desktop, when the user clicks on the "Next project" / "Previous project" buttons, they will navigate to the next/previous project details in the popup.
2. If the project is the first one, the "Previous project" button is disabled
3. If the project is the last one, the "Next project" button is disabled

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -110,22 +118,66 @@ <h4 class="backyear">Backend Dev</h4>
...
+      <div class="tonic-card-modal-container" id="tonic-card-modal-container">
+        <div class="tonic-card-modal">
+          <h1 class="tonic-title">Tonic Modal</h1>
+          <div class="tonic-stacks">
+            <ul>
+              <li class="tonic-stack1"></li>
+              <li class="tonic-stack2"></li>
+              <li class="tonic-stack3"></li>
+            </ul>
+          </div>
```
Comment (rotimiazeez) [inline index.html:143]: - [ ] Please make sure the popup follows the design for both desktop and mobile version from the chosen template in Figma (e.g., font, colours, images, tests, margins).
- Your design
screenshot (2)
- Figma design
screenshot (3)

File: index.html [hunk 2]
```diff
@@ -110,22 +118,66 @@ <h4 class="backyear">Backend Dev</h4>
...
+                ><span>See Live</span>
+                  <svg class="w-6 h-6" fill="none"
+                    stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+                      d="M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14"></path>
+                  </svg></a>
+                <a href="https://github.com/zemola/MY-PORTFOLIO.git" target="_blank" rel="noreferrer noopener">
+                  <span>See Source</span>
+                  <i class="fab fa-github-square network-link"></i>
+                </a>
```
Comment (rotimiazeez) [inline index.html:169]: Please make sure this also match your preferred template design✔👌
- Your Design
screenshot (4)
- Figma Design
screenshot (5)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @zemola 👋, 

### Good job so far!💯

### To highlight 👏
- Good UI 👌
- Good README 👍
- Descriptive PR message 😉
- No Linter error ✔
#### There are some issues that you still need to work on to go to the next project but you are almost there!
#### Just a few steps and you are unstoppable😎

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @zemola 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**